python/dri-migration/create_ingest_metadata.py

```diff
@@ -0,0 +1,109 @@
+import io
+import json
+import re
+from os import listdir
+import oracledb
+import os
+import boto3
+from botocore.config import Config
+import hashlib
+
+oracledb.init_oracle_client(lib_dir=os.environ['CLIENT_LOCATION'])
+conn = oracledb.connect(dsn='localhost/SDB4', user="STORE", password=os.environ['STORE_PASSWORD'])
+cur = conn.cursor()
+
+puid_lookup = {}
+
+
+def create_skeleton_suite_lookup(prefix):
+    path = f"{os.environ['DROID_PATH']}\\{prefix}"
+
+    pattern = re.compile(r'((x-)?fmt-\d{1,5})-signature-id-\d{1,5}(\.[A-Za-z]{1,10})')
+
+    directory_list = listdir(path)
+
+    for name in directory_list:
+        match = pattern.search(name)
+        if match:
+            puid = match.group(1).replace('fmt-', 'fmt/')
```

Comment:
Do you think it's worth doing this `.replace(f'{prefix}-', f'{prefix}/')` instead?